src/geophires_x/CylindricalReservoir.py

```diff
@@ -109,6 +114,7 @@ def __init__(self, model: Model):
         # internal values required for calculations
         self.depth = self.ParameterDict[self.depth.Name] = floatParameter(
             "Drilled length",
+            value=0.0,
```

Comment:
Remove - only `DefaultValue` should be specified, not `value` (which is redundant with `DefaultValue` in this case anyway)